catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -88,6 +91,10 @@
 
   private static final String QUOTE = "\"";
 
+  private static final String SUGGEST_QUERY_PARAM = "suggest.q";
```

Comment:
Can we access these : https://lucene.apache.org/solr/5_2_1/solr-core/org/apache/solr/spelling/suggest/SuggesterParams.html#SUGGEST_Q?